include/exadg/utilities/exadg_git_info.cpp

```diff
@@ -0,0 +1,41 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2021 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#include <exadg/utilities/exadg_git_info.h>
+#include <exadg/utilities/exadg_git_revision.h>
+
+namespace ExaDG
```

Comment:
Could we keep the interface similar is it in deal.II: https://github.com/dealii/dealii/blob/master/include/deal.II/base/revision.h.in?

Reply:
@peterrum I can do that, but I would like to keep the additional ```.cpp``` file. Otherwise, after every commit, there might be a lot of headers that have to be rebuilt. I can unify this, such that also the header of the ```deal.II``` revision is included in the source file.
Any comments? :)

Reply:
Sure :+1: